vms/spdagvm/vm.go

```diff
@@ -87,6 +87,7 @@ func (vm *VM) Initialize(
 	genesisBytes []byte,
 	toEngine chan<- common.Message,
 	fxs []*common.Fx,
+	txFee uint64,
```

Comment:
All DAG VMs must implement the [DAGVM](https://github.com/ava-labs/gecko/blob/master/snow/engine/avalanche/vm.go#L14) interface. So, [Initialize](https://github.com/ava-labs/gecko/blob/master/snow/engine/common/vm.go#L13-L38) can't have this added here.